src/main/resources/vm/script/datamanagement/refresh.vm

```diff
@@ -33,6 +34,10 @@ function refresh_token {
     fi    
     
     while :; do
+
+        # get the new refresh token
+        subshell_refresh_token=`cat $shanoir_refresh_token_location`
```

Comment:
Is `$shanoir_refresh_token_location` is different variable from the one in upper case ? where is it defined ?

Reply:
no it's the same i was just testing on my local and i forgot to change it. 
the loop proccess always uses the access token from the file. the bug was that it only reads the file the first time. even if the access token is refreshed, the loop still uses the old one. thats why i added this line.